buildspec/release/50githubrelease.yml

```diff
@@ -19,22 +19,24 @@ phases:
         commands:
             # Check for implicit env vars passed from the release pipeline.
             - test -n "${TOOLKITS_GITHUB_REPO_OWNER}"
+            - test -n "${TARGET_EXTENSION}"
             - REPO="${TOOLKITS_GITHUB_REPO_OWNER}/aws-toolkit-vscode"
 
     build:
         commands:
+            - echo "TARGET_EXTENSION=${TARGET_EXTENSION}"
             # pull in the build artifacts
             - cp -r ${CODEBUILD_SRC_DIR_buildPipeline}/* .
-            - VERSION=$(node -e "console.log(require('./packages/toolkit/package.json').version);")
-            - UPLOAD_TARGET=$(ls aws-toolkit-vscode*.vsix)
+            - VERSION=$(node -e "console.log(require('./packages/${TARGET_EXTENSION}/package.json').version);")
+            - UPLOAD_TARGET=$(ls *.vsix)
             - HASH_UPLOAD_TARGET=${UPLOAD_TARGET}.sha384
             - 'HASH=$(sha384sum -b $UPLOAD_TARGET | cut -d" " -f1)'
             - echo "Writing hash to $HASH_UPLOAD_TARGET"
             - echo $HASH > $HASH_UPLOAD_TARGET
             - echo "posting $VERSION with sha384 hash $HASH to GitHub"
             - RELEASE_MESSAGE="AWS Toolkit for VS Code $VERSION"
             - |
-                if [ $STAGE = "prod" ]; then
+                if [ $STAGE = "prod" || $TARGET_EXTENSION != 'toolkit' ]; then
```

Comment:
Do we need to revisit this later (i.e. is it in our checklist or backlog)?

Reply:
yes